docs/docs-content/automation/automation.md

```diff
@@ -0,0 +1,26 @@
+---
+sidebar_label: "Automation"
+title: "Automation"
+description: "Learn how to use automation tools with Palette and Palette VerteX."
+hide_table_of_contents: false
+sidebar_position: 0
+sidebar_custom_props:
+  icon: "terminal"
+tags: ["automation"]
+---
+
+This section contains documentation and guides essential for automating tasks with Palette. These resources cover the
+Palette CLI, Terraform provider, and Crossplane provider.
+
+The Palette CLI has multiple functionalities that allow users to interact with Palette and create and manage resources
+such as private cloud gateways. You can leverage the Palette CLI in Continuous Delivery/Continuous Deployment (CI/CD)
+pipelines and other automation tasks.
+
+Palette also supports Infrastructure as Code (IaC) tools that provide comprehensive automation for deploying and
+managing Palette resources. These resources include cluster profiles, cloud accounts, clusters, and more.
```

Comment:
```suggestion
Palette also supports Infrastructure as Code (IaC) tools and workflows.  We expose two providers to help you start managing Palette resources through automation. You can use our [Palette Terraform provider](./terraform/terraform.md) or our [Palette Crossplane provider](./crossplane/crossplane.md) to get started with IaC workflows today. Use either provider to manage Palette resources such as cluster profiles, cloud accounts, clusters, and more.
```

Take a peek and see if you like this intro.

Reply:
:white_check_mark: